language-plutus-core/language-plutus-core.cabal

```diff
@@ -314,4 +316,32 @@ benchmark language-plutus-core-budgeting-bench
         deepseq -any,
         lens -any,
         directory -any,
-        integer-gmp -any
+        integer-gmp -any,
+        hedgehog -any
+
+benchmark language-plutus-core-create-cost-model
```

Comment:
I don't know really. I put it here because it's rather module-internal rather than something you want to use as an end-user, so I figured `executable` wouldn't work too well.

Reply:
I *think* the only difference between an executable and a benchmark is that a benchmark gets run when you do `cabal bench` (seems you can `cabal run` benchmarks, so that's fine). I'm not sure that we want that, so I think it's fine for it to be an executable. Nobody is going to get confused, I don't think.

Reply:
Tested it, benchmark works nicely. Doesn't get run on `test`, and it gets the PWD right.